### ISSUE TITLE
suite used as `assert.WhatEver(suite.T(), ...`

### ISSUE THREAD

**Issue by ccoVeille** [Antonboom/testifylint#123]
Example

https://github.com/wanliu/go-oauth2-server/blob/f928fa1580dfcb03bfa77c2786980800f4af2c68/oauth/introspect_test.go#L152

```go
assert.NoError(suite.T(), err, "Inserting test data failed")
```

The pattern is very common
https://github.com/search?q=%22assert.NoError%28suite.T%28%29%22&type=code

**Comment by ccoVeille** [comment]
Do you agree it's something that should be suggested to fix ?

**Comment by Antonboom** [comment]
https://github.com/Antonboom/testifylint#suite-dont-use-pkg

**Comment by ccoVeille** [comment]
I'm so tired 😩 … I checked it in the unit tests of suite-subtest, which of course don't catch it. I should have launched testifylint directly 🤦

**Comment by ccoVeille** [comment]
So duplicate of #35, implemented in #104
